### ISSUE TITLE
False Error Reports

### ISSUE THREAD

**Issue by zeka0** [PyCQA/pylint#1490]
### Steps to reproduce
```python
import os
rootDir = '.'

for dirPath, dirNames, fileNames in os.walk(rootDir):
    for fileName in fileNames:
        if fileName.endswith('.pyc'):
            os.remove( os.path.join(dirPath, fileName) )
```

### Current behavior
![default](https://cloud.githubusercontent.com/assets/13997167/26238580/464b0c68-3cac-11e7-84b6-5cd1b5a6c277.PNG)

### Expected behavior
At line 3, declaration of variable 'rootDir' should be totally fine.
At line 8, the syntax is absolutely fine.

### pylint --version output
pylint-script.py 1.6.4, 
astroid 1.4.9
Python 3.5.2 |Anaconda custom (64-bit)| (default, Jul  5 2016, 11:41:13) [MSC v.1900 64 bit (AMD64)]

**Comment by rogalski** [comment]
Both of those warnings and correct and messages associated with them are self-explainatory.

Closed as not-a-bug.

**Comment by zeka0** [comment]
Alright, time to uninstall pylint.

**Comment by cpitclaudel** [comment]
@zeka0 I think this was the right response — pylint has a number of warnings that you might not agree with, but they can all be disabled, and the default config errs on the conservative side.  Each of them has a unique name (`bad-whitespace`, `invalid-name`), and mentioning that name in an appropriate exclude directive in a `.pylint` file is enough to disable them.

**Comment by zeka0** [comment]
Sir I didn't find any lines in pylint document mentioning '.pylint'. 
Therefore I can't be sure about: where to put it? How to tell pylint to look for it? etc...
The only thing I found is that they require you to add 1 line containing "#pylint disable-all" to the top of all modules, which I personally find very bothering.

**Comment by cpitclaudel** [comment]
@zeka0 https://stackoverflow.com/questions/4341746/how-do-i-disable-a-pylint-warning https://stackoverflow.com/questions/1372295/how-to-specify-a-configuration-file-for-pylint-under-windows http://code.litomisky.com/2014/10/24/getting-with-pylint-in-emacs/

**Comment by zeka0** [comment]
Finally disabled those warnings!
Cheers!
